nodes.tf

```diff
@@ -0,0 +1,50 @@
+resource "google_container_node_pool" "primary_nodes" {
+  provider = google-beta
+
+  name    = "${var.project_id}-node-pool${var.cluster_name_suffix}-${var.gke_instance_type}"
+  cluster = google_container_cluster.primary.name
+
+  project  = var.project_id
+  location = local.location
+
+  node_config {
+    oauth_scopes = var.oauth_scopes
```

Comment:
We should include the disk type as variable and set SSD and very low size 10GB as our MS are stateless, nothing but the code is stored

Reply:
Reading the K8s minimum requirements, maybe 10GB is too low
```Minimum 15 GB hard disk space for the file system containing```
We can set 20GB and compare the costs against 100GB non-ssd https://cloud.google.com/products/calculator to make the final decision

Reply:
As per the calculator, a 100Gb regional pd-standard disk cost U$D 8 a month, and a 20Gb regional pd-ssd disk costs U$D 6.80.

IMHO, I'll go with the 100Gb pd-standard disk, because we also need to take into account these variables:
* The number and size of the docker images stored in the node
* The size and rotation of the pod logs (despite if we're logging to stdout those logs are stored in the node)

Maybe we could shrink it to 80Gb that costs U$S 6,40 a month

Reply:
I like the default of 100 GB (which is I believe the value used by the GCP module.

Reply:
My intention was to improve the performance changing to SSDs, changing the size was just to reduce the cost of the SSD only
What would be the minimum disk to set for SSD? 40GB?
How it will compare in cost terms against the current solution (HDD-100GB)?